e2e_testing/torchscript/type_promotion.py

```diff
@@ -34,6 +34,51 @@ def TypePromotionSameCategoryDifferentWidthModule_basic(module, tu: TestUtils):
         torch.randint(10, [4]))
 
 
+class TypePromotionSameCategoryDifferentWidthMulModule(torch.nn.Module):
```

Comment:
The tests in this file are for testing type promotion infrastructure.  Mul and Div are ops using the infrastructure and need to be moved to basic.py.